### DISCUSSION TITLE
Build Samples with local FlatSharp

### DISCUSSION THREAD

**Discussion by joncham** [jamescourtney/FlatSharp#317] [General]
It seems like the Samples project always pulls the FlatSharp assemblies from NuGet. Is there a way to easily reference locally built assemblies out of the same source tree? Otherwise if I add a new sample relying on changes in the same PR it doesn't work. Thanks!

**Comment by jamescourtney** [comment]
Hi @joncham  -- Sorry I just saw this. You can do what the FlatSharp "E2E" tests do, which is to manually invoke the compiler as a prebuild step. Another thing I do is create a "local nuget" and put .nupkg files there, which VS can see if you add a local folder as a source.
